src/components/PlayerProfile.vue

```diff
@@ -20,25 +18,23 @@
                 loading="Loading Avatar..."
               >
               </v-img>
-              <h3 class="justify-center"
-                style="
-                text-align:center;
-                margin-top:10px"
-                v-if="player.connectedTo!=null"
-                >
-                This User Is Online On {{player.connectedTo.name}}.
-                </h3>
-                <v-divider block style="margin-top:2%;margin-bottom:2%"
-                 v-if="player.connectedTo!=null"></v-divider>
+              <v-divider block style="margin-top:2%;margin-bottom:2%"></v-divider>
               <div primary-title class="headline">{{player.name}}</div>
               <!-- <v-icon color="accent">perm_identity</v-icon> -->
               <h3 class="justify-center"
                   style="text-align:center;">
                 Name: {{player.lastSeenName}}
                 <br>UUID: {{player.uuid}}</h3>
               <v-divider block style="margin-top:2%;margin-bottom:2%"></v-divider>
-              <h3 class="justify-center"
-                style="text-align:center;">Rank(s): {{playerRanks}}</h3>
```

Comment:
Please keep the ranks heading. I think "Ranks" would be okay, we don't need to account for the singular case specifically.
Perhaps we can find a nicer way to display headings, not show them inline? Does Vuetify do something similar in the demos we could replicate here?